docs/source/cpp/parquet.rst

```diff
@@ -102,7 +102,8 @@ Encodings
 | DELTA_LENGTH_BYTE_ARRAY  | ✓        |          |         |
 +--------------------------+----------+----------+---------+
 
-* \(1) Only supported for encoding definition and repetition levels, not values.
+* \(1) Only supported for encoding definition, repetition levels,
```

Comment:
```suggestion
* \(1) Only supported for encoding definition and repetition levels,
```

(it's also "definition level", so I think it requires an "and" here to read correctly?)

Reply:
ah, okay. I went back and forth on that one, if those belonged together. Thank you.